argoverse/utils/transform.py

```diff
@@ -48,6 +55,13 @@ def quat_argo2scipy(q: np.ndarray) -> np.ndarray:
     return q_scipy
 
 
+def quat_scipy2argo(q: np.ndarray) -> np.ndarray:
```

Comment:
This is on a bit of a meta note, but we really need to clean up `transform.py` (the file as a whole).